axelor-base/src/main/resources/domains/AppBase.xml

```diff
@@ -20,6 +20,9 @@
     <!-- Interfaces QAS et SVI -->
     <string name="qasWsdlUrl" title="URL to QAS WSDL"/>
 
+
+    <string name="draftPrefix" title="Draft prefix"/>
```

Comment:
```suggestion
    <string name="draftPrefix" title="Draft prefix" default="#"/>
```
Just a small suggestion because I think it would be a bit more user friendly this way. Otherwise, the implementation seems fine, good job! 👍🏽